README.md

```diff
@@ -1 +1,65 @@
-# conclave-cii-api
\ No newline at end of file
+# CONCLAVE CII API
+This is the Conclave Central Identity Index (CII) API service.
+
+## Nomenclature
+
+- **OrganisationSchemeIdentifier**: identifier used for an organisation with a scheme
+- **Scheme**: 
+- **SchemeRegister**: 
+
+## Technical documentation
+
+This is a Ruby on Rails application that .... It's only presented as an internal API and doesn't face public users.
```

Comment:
@ccsnareshkumar feel free to fill in the blanks here

Reply:
@iceraluk filled